templates/primary-secondary/test/primary-secondary.visual-diff.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+	<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+	<script type="module">
+		import '../../../components/typography/typography.js';
+		import '../primary-secondary.js';
+	</script>
+	<title>d2l-template-primary-secondary</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<meta charset="UTF-8">
+</head>
+<body class="d2l-typography">
+	<div class="visual-diff">

```

Comment:
The `visual-diff` CSS class just puts some margins below the `<div>` so that when we're testing many things on a page they don't get bunched up -- I think it can be excluded.

Also `styles.css` puts some padding on the body that you likely don't want, so I think the whole CSS file can be left out.

I'm wondering since these templates should only be used one-per-page, if this should be broken out into one file per visual diff test? Otherwise They could be interfering with each other, especially once we start doing responsive stuff with the panels.